doc/connectivity/usb/device/usb_device.rst

```diff
@@ -579,8 +579,6 @@ The following Product IDs are currently used:
 +----------------------------------------------------+--------+
 | :zephyr:code-sample:`usb-cdc-acm-composite`        | 0x0002 |
 +----------------------------------------------------+--------+
-| :zephyr:code-sample:`usb-hid-cdc`                  | 0x0003 |
```

Comment:
The PIDs cannot be reused and therefore this should be marked as Reserved (previously: usb-hid-cdc).